src/features/Apiexplorer/index.tsx

```diff
@@ -20,7 +20,7 @@ export default function ApiExplorerFeatures() {
   const has_info = Object.keys(request_info).length === 0;
```

Comment:
Can we rename the folder `Apiexplorer` to `ApiExplorer`?

Reply:
@prince-deriv Yes, We can but this will add more file changes due to folder name changes which I think we shouldn't do that we can't do it later.